torchaudio/csrc/rnnt/autograd.cpp

```diff
@@ -16,7 +16,7 @@ class RNNTLossFunction : public torch::autograd::Function<RNNTLossFunction> {
       double clamp,
       bool fused_log_smax = true,
       bool reuse_logits_for_grads = true) {
-    at::AutoNonVariableTypeMode g;
+    at::AutoDispatchBelowADInplaceOrView g;
```

Comment:
You should not need to use this guard in custom Function. The forward already runs with autograd disabled.

Reply:
Thanks Alban, that makes total sense to me - I just figured there must be some deep reason I don't understand. Removed it.